modules/Emails/mail.php

```diff
@@ -307,29 +307,48 @@ function setMailerProperties($mail, $subject, $contents, $from_email, $from_name
   * $mail -- reference of the mailobject
   */
 function setMailServerProperties($mail) {
-	global $adb,$default_charset;
+	global $adb,$default_charset, $current_user;
 	$adb->println('> setMailServerProperties');
-
+	$user_mail_config = $adb->pquery('select * from vtiger_mail_accounts where user_id=?', array($current_user->id));
```

Comment:
you are not checking that the outgoing email server has been configured. Imagine a user who configures incoming email server but does not configure outgoing email server. In that case, there will be one row returned but not valid outgoing email server settings.   Am I correct?

Reply:
absolutely correct